images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -0,0 +1,221 @@
+param (
+    [Parameter(Mandatory)][string]
+    $OutputDirectory
+)
+
+# Install MarkdownPS module for software report generation
+Install-Module MarkdownPS -Force -Scope CurrentUser
```

Comment:
I think we already install this module during image generation so no need to install it again

Reply:
This approach uses for Windows images not for Ubuntu, where powershell modules are installed using toolset.json.

Reply:
I see, thanks for explanation